lib/msal-browser/package.json

```diff
@@ -90,6 +90,6 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "@azure/msal-common": "^10.0.0"
+    "@azure/msal-common": "file:../msal-common/dist"
```

Comment:
We do this so that people can take advantage of new versions of common as they come out without needing to update to the latest msal-browser

Reply:
Ok.  I'll rollback this change and see if things build.... i think we'll get a conflict in node_modules, but we'll see.

Reply:
Fixed.